datafusion/src/physical_plan/planner.rs

```diff
@@ -866,517 +865,487 @@ impl DefaultPhysicalPlanner {
             exec_plan
         }.boxed()
     }
+}
 
-    /// Create a physical expression from a logical expression
-    pub fn create_physical_expr(
-        &self,
-        e: &Expr,
-        input_dfschema: &DFSchema,
-        input_schema: &Schema,
-        ctx_state: &ExecutionContextState,
-    ) -> Result<Arc<dyn PhysicalExpr>> {
-        match e {
-            Expr::Alias(expr, ..) => Ok(self.create_physical_expr(
-                expr,
-                input_dfschema,
-                input_schema,
-                ctx_state,
-            )?),
-            Expr::Column(c) => {
-                let idx = input_dfschema.index_of_column(c)?;
-                Ok(Arc::new(Column::new(&c.name, idx)))
-            }
-            Expr::Literal(value) => Ok(Arc::new(Literal::new(value.clone()))),
-            Expr::ScalarVariable(variable_names) => {
-                if &variable_names[0][0..2] == "@@" {
-                    match ctx_state.var_provider.get(&VarType::System) {
-                        Some(provider) => {
-                            let scalar_value =
-                                provider.get_value(variable_names.clone())?;
-                            Ok(Arc::new(Literal::new(scalar_value)))
-                        }
-                        _ => Err(DataFusionError::Plan(
-                            "No system variable provider found".to_string(),
-                        )),
+/// Create a physical expression from a logical expression ([Expr])
```

Comment:
again, whitespace blind diff is the best to see -- this change makes this a free function and changes the parameters from 
```rust
        ctx_state: &ExecutionContextState,
```

to 

```rust
    execution_props: &ExecutionProps,
```